### PR TITLE
Likes and bookmarks

### PR DESCRIPTION
**Pull request by gopalbbharadva**
### PR for Likes and Bookmarks.

-> Like/ unlike to any post.
-> add/remove post to/from bookmark page.

live link: https://deploy-preview-11--sm-spotter.netlify.app/

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]
- code indentation is well
- code written well
- functionality working fine

File: src/features/bookmarkSlice.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    } catch (error) {
+      return rejectWithValue({ message: "Rmove post from bookmark failed" });
+    }
+  }
+);
+
+const bookmarkSlice = createSlice({
+  name: "bookmark",
+  initialState,
+  extraReducers: (builder) => {
```
Comment (VedanthB) [inline src/features/bookmarkSlice.js:54]: you can also do this, extraReducers: [getBookMarkPosts.pending] : (state, {payload}) => {}[getBookMarkPosts.rejected] : (state, {payload}) => {}

doesnt make a difference tho

File: src/features/postSlice.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
```
Comment (VedanthB) [inline src/features/postSlice.js:2]: i don't think you are using axios here as all the api calls are in the services folder

**Review by VedanthB — APPROVED** [review]
Good work Gopal,

I like that you have used tailwind and the ui looks really nice. The code is clean and easy to understand.

You can merge the PR

File: src/features/bookmarkSlice.js (hunk 1 above)
Comment (gopalbbharadva) [reply src/features/bookmarkSlice.js:54]: > you can also do this, extraReducers: [getBookMarkPosts.pending] : (state, {payload}) => {}[getBookMarkPosts.rejected] : (state, {payload}) => {}
> 
> doesnt make a difference tho

ok, I will remove it in next PR.

File: src/features/postSlice.js (hunk 2 above)
Comment (gopalbbharadva) [reply src/features/postSlice.js:2]: Ok i will remove it in next PR.

**Pull request merged**